src/RenameCodeActionProvider.ts

```diff
@@ -0,0 +1,30 @@
+
+import * as vscode from 'vscode';
+import { DefinitionKind } from './INamedDefinition';
+import { PositionContext } from './PositionContext';
+const COMMAND = 'editor.action.rename';
```

Comment:
nit: command would be our standard naming

Reply:
Fixed.